### PR TITLE
Adding Lumin Platform to XR

### PR DESCRIPTION
**Pull request by StayTalm**
Lumin is the Magic Leap device.  
Adding in that platform as a supported XR platform.  
Also adding a secondary define that can be used by project to force enable the XR platform (for NDA and prototype platforms.

Tested on Magic Leap headset.

### CODE REVIEW

**Review by tropicdragon — APPROVED** [review]
I assume at some point we should have a catch-all define for XR platforms?

File: Packages/com.unity.inputsystem/CHANGELOG.md [hunk 1]
```diff
@@ -97,6 +99,7 @@ however, it has to be formatted properly to pass verification tests.
   ```
   - Added support for Step Counter sensors for iOS.
     * You need to enable **Motion Usage** under Input System settings before using the sensor. You can also manually add **Privacy - Motion Usage Description** to your application's Info.plist file.
+  - Add Lumin
```
Comment (jimon) [inline Packages/com.unity.inputsystem/CHANGELOG.md]: Maybe "Added support for Magic Leap Lumin OS" suits better?

File: Packages/com.unity.inputsystem/CHANGELOG.md [hunk 2]
```diff
@@ -37,6 +37,8 @@ however, it has to be formatted properly to pass verification tests.
   * This is to address the problem of mouse input leading to __both__ mouse and touch input happening concurrently. Instead, enabling touch simulation will now effectively __replace__ mouse and pen input with touch input.
   * Devices such `Mouse` and `Pen` will remain in place but will not get updated. Events received for them will be consumed by `TouchSimulation`.
 - Enabled XR device support on Switch.
+- Enabled XR device support on Lumin (Magic Leap).
```
Comment (jimon) [inline Packages/com.unity.inputsystem/CHANGELOG.md]: Maybe "Enabled XR device support on Magic Leap Lumin OS" suits better?

File: Packages/com.unity.inputsystem/CHANGELOG.md (hunk 1 above)
Comment (jimon) [reply Packages/com.unity.inputsystem/CHANGELOG.md]: Fixed: I've removed it

File: Packages/com.unity.inputsystem/CHANGELOG.md (hunk 2 above)
Comment (jimon) [reply Packages/com.unity.inputsystem/CHANGELOG.md]: Fixed: rephrased it

**Review by jimon — APPROVED** [review]

**Comment by caryhaynie** [comment]
@jimon @Rene-Damm what version of the package is this expected to land?

**Pull request merged**